scripts/export-to-pdf-fix/on-create.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+# OVERVIEW
+# This script fixes a problem in SageMaker Notebook instances where Jupyter fails to export a notebook
+# directly to PDF. nbconvert depends on XeLaTeX and several LaTeX packages that are non-trivial to
+# install because `tlmgr` is not included with the texlive packages provided by yum.
+
```

Comment:
We should explicitly document that this required internet access to fetch the binaries from the `ctan`

Reply:
Done.